src/stdlib/SDL_iconv.c

```diff
@@ -31,6 +31,10 @@
 #include "SDL_endian.h"
 
 #if defined(HAVE_ICONV) && defined(HAVE_ICONV_H)
+#ifdef __FreeBSD__
+/* Define LIBICONV_PLUG to use iconv from the base instead of ports and avoid linker errors. */
```

Comment:
This part doesn't belong here - you already made a PR for that.

Reply:
Fixed.